students/10th/taehee choi/westagram/js/login.js

```diff
@@ -0,0 +1,17 @@
+const idInput = document.getElementsByClassName("login_id")[0]; //.value method는 string 타입으로 변환됨 
+const pwInput = document.getElementsByClassName("login_pw")[0];
+const btn = document.getElementsByClassName("login_button")[0];
+
+idInput.addEventListener("keyup", changeColor);
+pwInput.addEventListener("keyup", changeColor);
+
+
+function changeColor() {
+    if (idInput.value !== "" && pwInput.value !== "") {
+        btn.style.backgroundColor = "green";
+        document.querySelector(".login_button").removeAttribute("disabled");
+    }
+    else {
+        btn.style.backgroundColor = "lightblue";
+    }
```

Comment:
```suggestion
    } else {
        btn.style.backgroundColor = "lightblue";
    }
```

if 문의 else if나 else 는 이런 식으로 이어서 작성해주세요.